code/datums/diseases/advance/symptoms/organs.dm

```diff
@@ -0,0 +1,222 @@
+				M.set_blindness(0)
+				M.set_blurriness(0)
+			else if(eyes.damage > 0)
+				eyes.applyOrganDamage(-1)
+		else
+			if(prob(base_message_chance))
+				to_chat(M, "<span class='notice'>[pick("Your eyes feel great.","You feel like your eyes can focus more clearly.", "You don't feel the need to blink.","Your ears feel great.","Your healing feels more acute.")]</span>")
+
+
+/datum/symptom/organ_restoration //heals damage to other internal organs that get damaged far less often
+	name = "Organ Restoration"
+	desc = "The virus stimulates rapid cell growth in organ tissues, slowly repairing the host's organs over time."
+	stealth = 2
+	resistance = 3
+	stage_speed = -2
+	transmission = -1
+	level = 6
+	severity = -1
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	prefixes = list("Organ 9")
+	var/curing = FALSE
+	var/regenorgans = FALSE 
+	threshold_desc = "<b>Stealth 4:</b> The host will regenerate missing organs over a long period of time.<br>\
+					  <b>Stage Speed 10:</b> The virus causes the host's internal organs to gain some self-correcting behaviour, preventing heart attacks and appendicitis.<br>"
+
+/datum/symptom/organ_restoration/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.stealth >= 4)
+		severity -= 1
+
+/datum/symptom/organ_restoration/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.stage_rate >= 10)
+		curing = TRUE
+	if(A.stealth >= 4)
+		regenorgans = TRUE
+
+/datum/symptom/organ_restoration/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	var/status = ORGAN_ORGANIC
+	if(MOB_ROBOTIC in A.infectable_biotypes)
+		status = null //if the disease is capable of interfacing with robotics, it is allowed to heal mechanical organs
+	if(A.stage >= 4)
+		M.adjustOrganLoss(ORGAN_SLOT_APPENDIX, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_STOMACH, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_LUNGS, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_HEART, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_LIVER, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_TAIL, -1, required_status = status)
+		M.adjustOrganLoss(ORGAN_SLOT_WINGS, -1, required_status = status)
+		if(curing)
+			for(var/datum/disease/D in M.diseases)
+				if(istype(D, /datum/disease/appendicitis) || istype(D, /datum/disease/heart_failure))
+					D.cure()
+			if(M.undergoing_cardiac_arrest())
+				M.set_heartattack(FALSE)
+		if(regenorgans) //regenerate missing organs that this disease cures, other than the brain
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				var/datum/species/S = H.dna.species
+				if(!M.getorgan(/obj/item/organ/appendix) && !((TRAIT_NOHUNGER in S.inherent_traits) || (TRAIT_POWERHUNGRY in S.inherent_traits)))
+					var/obj/item/organ/appendix/O = new()
+					O.Insert(M)
+					M.adjustOrganLoss(ORGAN_SLOT_APPENDIX, 199, 199) //don't make it fail, or the host will start taking massive damage
```

Comment:
this is somehow letting people have a failed appendix again and again. I think it needs to be a little bit lowered.

Reply:
figured out the issue